lib/RTCSession.js

```diff
@@ -504,20 +496,22 @@ module.exports = class RTCSession extends EventEmitter
   /**
    * Answer the call.
    */
-  answer(options = {})
+  answer(mediaConnection, options = {})
   {
     debug('answer()');
 
+    if (!isMediaConnection(mediaConnection)) {
+      throw new Exceptions.InvalidMediaConnectionError('Invalid MediaConnection implementation!')
+    }
+
+    this._connection = mediaConnection;
+
     const request = this._request;
     const extraHeaders = Utils.cloneArray(options.extraHeaders);
     const mediaConstraints = Utils.cloneObject(options.mediaConstraints);
```

Comment:
`mediaConstraints` where used for `getUserMedia()`. Which is not used anymore. Please remove it and all its references.